### PR TITLE
DAOS-7128 test: Add a test for container copy.

### PR DESCRIPTION
**Pull request by ashleypittman**
Add a NLT test to copy a container.
Parallel-build: true

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

File: utils/node_local_test.py [hunk 1]
```diff
@@ -1478,6 +1481,27 @@ def test_uns_basic(self):
         if dfuse.stop():
             self.fatal_errors = True
 
+    def test_cont_copy(self):
```
Comment (daltonbohning) [inline utils/node_local_test.py:1484]: Minor suggestion - only if you need to push again:
There are two tools:
```
daos container copy # Copy cont->cont with Obj API
daos filesystem copy # Copy cont<->POSIX with DFS API
```
So the name `test_cont_copy` might be confusing if ever there is a need to to add a `daos container copy` test here.

**Review by daltonbohning — APPROVED** [review]
LGTM. Just a minor suggestion.

**Review by daltonbohning — APPROVED** [review]
LGTM

File: utils/node_local_test.py (hunk 1 above)
Comment (dsikich) [reply utils/node_local_test.py:1484]: @daltonbohning @ashleypittman the Obj API version should be 'daos cont clone', so we could just use that as the name.

**Review by dsikich — APPROVED** [review]

File: utils/node_local_test.py (hunk 1 above)
Comment (ashleypittman) [reply utils/node_local_test.py:1484]: I'lll add that in a future PR.

**Pull request merged**